### PR TITLE
shcopyfoundburrowlocations

### PR DESCRIPTION
**Pull request by hannibal002**
/shcopyfoundburrowlocations

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/config/Storage.java [hunk 1]
```diff
@@ -468,6 +468,8 @@ public static class DianaStorage {
             // TODO renmae
             public MythologicalCreatureTracker.Data mythologicalMobTracker = new MythologicalCreatureTracker.Data();
 
+            @Expose
+            public List<LorenzVec> foundBurrowLocations = new ArrayList<>();
```
Comment (CalMWolfs) [inline src/main/java/at/hannibal2/skyhanni/config/Storage.java]: Should this be in profile specific or just in general, in general makes more sense to me, also if it is expected to get very long I feel it should be separated to a new config file

**Review by CalMWolfs — CHANGES_REQUESTED** [review]
I think that is the only thing thing that needs addressing

File: src/main/java/at/hannibal2/skyhanni/config/Storage.java (hunk 1 above)
Comment (hannibal002) [reply src/main/java/at/hannibal2/skyhanni/config/Storage.java]: Done.
When the list gets too long, we can write a smart "extract on file io actions" logic without damaging the usage of the config structure. (Something we should have used for known features and Jacob contests, and friends as well)

**Review by CalMWolfs — APPROVED** [review]
Code looks good to me

**Pull request merged**